File: Event Server/dreamE/server/datastore/category.go
package datastore

type Category struct {
	Id           int    `json:"id"`
	CategoryName string `json:"category_name"`
	Image        string `json:"image"`
}

// Category constructor
func NewCategory(id int, cat, img string) *Category {
	return &Category{
		Id:           id,
		CategoryName: cat,
		Image:        img,
	}
}

// Set Category ID
func (c *Category) SetID(id int) {
	c.Id = id
}

// Set Category Name
func (c *Category) SetCatName(cat string) {
	c.CategoryName = cat
}

// Set Category Image
func (c *Category) SetImage(img string) {
	c.Image = img
}

// Get Category Id
func (c *Category) GetID() int {
	return c.Id
}

// Get Category Name
func (c *Category) GetName() string {
	return c.CategoryName
}

// Get Category Image
func (c *Category) GetImage() string {
	return c.Image
}

File: Event Android/Event Main Android Studio/app/src/main/java/com/events/model/Category.java
package com.events.model;

import com.google.gson.annotations.SerializedName;

/**
 * Create   `AQ `d by Cruze on 12/7/2016.
 */

public class Category {
    @SerializedName("category_name")
    private String categoryName;
    private String image;
    private int id;

    public Category(int id, String categoryName, String image) {
        this.id = id;
        this.categoryName = categoryName;
        this.image = image;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getCategoryName() {
        return categoryName;
    }

    public void setCategoryName(String categoryName) {
        this.categoryName = categoryName;
    }

    public String getImage() {
        return image;
    }

    public void setImage(String image) {
        this.image = image;
    }
}
